src/Header.tsx

```diff
@@ -112,13 +124,37 @@ function HeaderContent({ anchor }) {
       <Tab value="" component={RouterLink} to="/" label={<Typography variant="h6" sx={{ px: 1 }}>
         <Trans t={t} i18nKey="pageTitle">Genshin Optimizer</Trans>
       </Typography>} />
-      {content.map(({ i18Key, value, to, icon, resize }) => <Tab key={value} value={value} component={RouterLink} to={to} icon={icon} iconPosition="start" label={(isLarge || !resize) && t(i18Key)} />)}
+      {content.map(({ i18Key, value, to, icon, resize }) =>
+        <Tab
+          key={value}
+          value={value}
+          component={RouterLink}
+          to={to}
+          icon={icon}
+          iconPosition="start"
+          label={(isLarge || !resize) && t(i18Key)}
+        />
+      )}
       <Box display="flex" alignItems="center">
         <Chip color="success" label={name} />
       </Box>
 
       <Box flexGrow={1} />
-      {links.map(({ i18Key, href, label, icon }) => <Tab key={label} component="a" href={href} target="_blank" icon={icon} iconPosition="start" onClick={e => ReactGA.outboundLink({ label }, () => { })} label={isLarge && t(i18Key)} />)}
+      {links.map(({ i18Key, href, label, icon }) =>
```

Comment:
Implement the patch link as a separate Component, instead of jerryrigging it into existing button flow(its behaviour is too complex)

Put the patch notes "seen" handling into this component instead of scattering it in several places

Reply:
Also, does it make sense to use up valuable navbar realestate? Don't we already display the version number on the site? why not make that the link

Reply:
I didn't like it in the footer because it's not super visible (have to scroll to the bottom of the site and not many people are there). The header is a bit tight squeezed, but I prefer that than putting it in the header. I could make the patch notes an icon-only (with tooltip) no matter what if that is any better

Reply:
There might be something in the future that will use up that top right space of navbar, so im hesitant to add anything emergent there. I think patchnotes on the home page, and the version number as a link is sufficient. most of the "new" features are easily apparent anyways.